### PR TITLE
Update `People` index and `Main_page` with new articles

### PR DESCRIPTION
**Pull request by Walavouchey**
This updates the people index page with the tournament and pp committee articles, and adds them to the main page

I've updated the french and indonesian translations along the way, and outdated the rest

I've also moved osu! UCI article in the main page to the Community section and put a link to the People index page. Not sure on the ordering.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/People/en.md [hunk 1]
```diff
@@ -4,5 +4,7 @@ Index page for articles about people or groups.
 
 - [Community Contributors](Community_Contributors)
 - [The Team](The_Team)
+- [Tournament Committee](Tournament_Committee)
 - [peppy](peppy)
+- [Performance Points Committee](Performance_Points_Committee)
```
Comment (TicClick) [inline wiki/People/en.md:9]: didn't you want to have the list sorted?

**Review by TicClick — DISMISSED** [review]

File: wiki/People/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/People/en.md:9]: woops

File: wiki/Main_Page/en.md [hunk 2]
```diff
@@ -102,18 +102,18 @@ Social interaction is one of main things that fuel osu! players all around the w
...
+# [People](/wiki/People)
 
 osu! wouldn't have been possible without many users helping with development, maintenance, and community management: their efforts, time, and dedication make the game live and prosper.
 
 [The Team](/wiki/People/The_Team): [Developers](/wiki/People/The_Team/Developers) • [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) • [Support Team](/wiki/People/The_Team/Support_Team) • [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) • [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) • [osu! Alumni](/wiki/People/The_Team/osu!_Alumni) • [Project Loved Team](/wiki/People/The_Team/Project_Loved_Team)
 
-Organisations: [osu! UCI](/wiki/Community/Organisations/osu!_UCI)
-
-[Community Contributors](/wiki/People/Community_Contributors) • [Users with unique titles](/wiki/People/Users_with_unique_titles)
+[Community Contributors](/wiki/People/Community_Contributors) • [Performance Points Committee](/wiki/People/Performance_Points_Committee) • [Tournament Committee](/wiki/People/Tournament_Committee) • [Users with unique titles](/wiki/People/Users_with_unique_titles)
```
Comment (Walavouchey) [inline wiki/Main_Page/en.md:116]: @TicClick opinion?

some translators seem to be sorting index pages as they go recently (i think), though this page obv doesn't have translators with their own sortings. maybe the main page should have fixed ordering regardless of translations (it's easier to navigate sorted lists, but the main page is just a big cloud of links).

either way i'd keep the sorting on the translations equal to the english ver., but do you have any opinions on the ordering on this?

Comment (TicClick) [reply wiki/Main_Page/en.md:116]: for regular index pages, such as one for disambiguation articles, it makes sense to sort the articles lexicographically, as it's more natural and they don't have any other clear ordering.

last time I worked on the main page, I've tried to order pages by their "importance", or "significance", so I'd say make the translations follow the English page

Comment (Walavouchey) [reply wiki/Main_Page/en.md:116]: alright so which do you think are the important ones here? maybe `Community Contributors` --> `Users with unique titles` --> `Tournament Committee` --> `Performance Points Committee`?

Comment (TicClick) [reply wiki/Main_Page/en.md:116]: anything will do, your suggestion also works (I'd make it roughly the same)

**Review by TicClick — APPROVED** [review]

**Pull request merged**